src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st

```diff
@@ -0,0 +1,63 @@
+"
+Description
+--------------------
+
+I am a command available in the class context menu of Calypso. When activated I'll jump on the test class corresponding to the class. 
+
+If the class does not exists I will create it in the corresponding package.
+	
+Internal Representation and Key Implementation Points.
+--------------------
+
+    Instance Variables
+	testClass:		<aClass>		Test class corresponding to the class I was activated with
+"
+Class {
+	#name : #ClyJumpToTestClassCommand,
+	#superclass : #SycSingleClassCommand,
+	#traits : 'TClyGenerateTestClass',
+	#classTraits : 'TClyGenerateTestClass classTrait',
+	#instVars : [
+		'browser',
+		'systemEnvironment'
+	],
+	#category : #'Calypso-SystemPlugins-SUnit-Browser-Commands'
+}
+
+{ #category : #activation }
+ClyJumpToTestClassCommand class >> fullBrowserMenuActivation [
+	<classAnnotation>
+	
+	^ CmdContextMenuActivation for: ClyClassContextOfFullBrowser
+]
+
+{ #category : #activation }
+ClyJumpToTestClassCommand class >> fullBrowserShortcutActivation [
+	<classAnnotation>
+	
+	^ CmdShortcutActivation by: $g meta, $j meta for: ClyClassContextOfFullBrowser
+]
+
+{ #category : #accessing }
+ClyJumpToTestClassCommand >> defaultMenuItemName [
+	^ 'Jump to test class'
+]
+
+{ #category : #execution }
+ClyJumpToTestClassCommand >> execute [
+	[ browser showClass: (self testClassFor: targetClass) ]
```

Comment:
and here it should be "browser selectClass:"

Reply:
Also there is special  exception CmdCommandAborted. You can always signal it from any command method. And existing users already handle it in safe way.
Not sure if your special exception should be replaced by this. So it's just for note. But think that current code will eat generation problems. Any command user will continue successful execution which is wrong. Abort signal should be explicitly handled to be ignored.